docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -305,6 +305,111 @@ document
 
 </PlatformSection>
 
+### Loading Strategies
+
+Because the `feedbackIntegration` is a user-facing integration, we offer two loading strategies that have implications on bundle size.
+
+<Note>
+For most users, we recommend using `feedbackIntegration` in your `Sentry.init` call. This will set up user feedback with good defaults matching the environment.
+</Note>
+
+All of the code examples on this page use `feedbackIntegration` as a default because it is available whether you have chosen the CDN or NPM installation method. However, the implementation of `feedbackIntegration` is different depending on the installation method. For NPM users `feedbackIntegration` is an alias of `feedbackSyncIntegration`. For CDN users, `feedbackIntegration` is an alias of `feedbackAsyncIntegration`.
```

Comment:
```suggestion

All of the code examples on this page use `feedbackIntegration` as a default because it's available regardless of whether you've chosen the CDN or NPM installation method. However, the implementation of `feedbackIntegration` is different for the two installation methods. For NPM users, `feedbackIntegration` is an alias of `feedbackSyncIntegration`. For CDN users, `feedbackIntegration` is an alias of `feedbackAsyncIntegration`.
```